test/e2e/storage/vsphere_volume_diskformat.go

```diff
@@ -75,14 +75,14 @@ var _ = framework.KubeDescribe("Volume Disk Format [Feature:vsphere]", func() {
 		if !isNodeLabeled {
 			nodeLabelValue := "vsphere_e2e_" + string(uuid.NewUUID())
 			nodeKeyValueLabel = make(map[string]string)
-			nodeKeyValueLabel["vsphere_e2e_label"] = nodeLabelValue
-			framework.AddOrUpdateLabelOnNode(client, nodeName, "vsphere_e2e_label", nodeLabelValue)
+			nodeKeyValueLabel["vsphere_e2e_label_volume_diskformat"] = nodeLabelValue
```

Comment:
Or you can use `f.Namespace.Name`  instead of hardcoded string.

Reply:
Thanks @pshahzeb for the review. I will make sure to generate PR against kubernetes master branch with labels created using namespace name.